bokeh/embed/elements.py

```diff
@@ -22,7 +22,17 @@
 
 # Standard library imports
 from html import escape
-from typing import List, Optional
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    Dict,
+    List,
+    Tuple,
+)
+
+## External imports
```

Comment:
This doesn't work, because isort will remove this header altogether and merge both sections. Sadly it doesn't understand type import pattern (it needs at least one import at the top-level). This needs to be kept until isort is improved.